### PR TITLE
Add docs for scheduled export of contacts

### PR DESCRIPTION
**Pull request by RCheesley**
It turns out nobody ever wrote the docs for https://github.com/mautic/mautic/pull/11272 which has resulted in some confusion (see https://forum.mautic.org/t/bulk-contact-export-email-does-not-arrive/31169/).

This PR adds the missing documentation in the configuration section (to turn on/off the background sending of exports) and also for the command line syntax/cron job information.

### CODE REVIEW

File: docs/configuration/settings.rst [hunk 1]
```diff
@@ -319,7 +319,7 @@ See :ref:`here<contact's unsubscribe email preferences>` to set the Contact's Em
 Default frequency rule
 ======================
 
-* **Do not contact more than <number> each <period>** - This limits the number of Marketing Messages a Contact receives in a certain period of time day, week, month. Transactional messages don't count towards this limit. You can adjust this at the individual Contact level, either manually or by Preference Center setting.
+* **Do Not Contact more than <number> each <period>** - This limits the number of Marketing Messages a Contact receives in a certain period of time day, week, month. Transactional messages don't count towards this limit. You can adjust this at the individual Contact level, either manually or by Preference Center setting.
```
Comment (RCheesley) [inline docs/configuration/settings.rst:322]: This was a red warning so I fixed it while I was here.

**Review by uadf — APPROVED** [review]
Nice.
Spent a lot of time this morning try to get my CSV export...

**Comment by RCheesley** [comment]
Thanks for the review @uadf 🚀

**Comment by RCheesley** [comment]
@all-contributors please add @uadf for review

**Pull request merged**